mfhdf/nctest/slabs.c

```diff
@@ -44,386 +44,326 @@ static void
 val_stuff(nc_type type, void *v, int ii, long val) /* v[ii] = val */
 {
     static char pname[] = "val_stuff";
-#ifdef WRONG_for_PGCC /* This way caused a lot of problems for PGI CC compiler                               \
-                         EIP 2004/12/15 */
-    union gp {
-        char   cp[1];
-        short  sp[1];
-        nclong lp[1];
-        float  fp[1];
-        double dp[1];
-    } * gp;
-
-    gp = (union gp *)v;
```

Comment:
This file had a LOT of changes due to the formatting changing after I removed the dead code. The only real changes I made to this file was adding the compare macros in a couple of places.